source/tyr/tyr/tasks.py

```diff
@@ -146,28 +124,78 @@ def import_data(
         # currently the name of a dataset is the path to it
         dataset.name = filename
         models.db.session.add(dataset)
+
+        # Create a job by type of data
+        job = models.Job()
+        job.instance = instance
+        job.state = 'pending'
         job.data_sets.append(dataset)
 
+        if new_action:
+            models.db.session.add(job)
+            models.db.session.commit()
+            # We pass the job id to each tasks, but job need to be commited for having an id
+            new_action.kwargs['job_id'] = job.id
+            actions.append(new_action)
+            actions.append(finish_job.si(job.id))
+
     if actions:
+        # Create a new job and dataset for ed2nav that will generate binary file (New .nav.lz4)
+        dataset = models.DataSet()
+        dataset.type = dataset.family_type = 'nav'
+        dataset.name = instance_config.target_file
+        models.db.session.add(dataset)
+
+        job = models.Job()
+        job.instance = instance
+        job.state = 'pending'
+        job.data_sets.append(dataset)
         models.db.session.add(job)
         models.db.session.commit()
-        # We pass the job id to each tasks, but job need to be commited for having an id
-        for action in actions:
-            action.kwargs['job_id'] = job.id
-        # Create binary file (New .nav.lz4)
+
         binarisation = [ed2nav.si(instance_config, job.id, custom_output_dir)]
         actions.append(chain(*binarisation))
+        actions.append(finish_job.si(job.id))
+
         # Reload kraken with new data after binarisation (New .nav.lz4)
         if reload:
             actions.append(reload_data.si(instance_config, job.id))
 
+        # Retrieve jobs created to pass dataset to "send_to_mimir"
         if not skip_mimir:
-            for dataset in job.data_sets:
-                actions.extend(send_to_mimir(instance, dataset.name, dataset.family_type))
-        else:
-            current_app.logger.info("skipping mimir import")
+            for job in models.Job.query.filter_by(state='pending').all():
```

Comment:
This will create new job for every job no finished yet!! Keep every job created if you must

Reply:
job_state is an Enum:
https://github.com/CanalTP/navitia/blob/a5fc0a11ce37d74ee5619f3b757cb2cd5e2d2a8d/source/navitiacommon/navitiacommon/models/__init__.py#L749
So 'created' should be added.